[2025-02-19T09:38:34.414+0000] {local_task_job_runner.py:123} INFO - ::group::Pre task execution logs
[2025-02-19T09:38:34.429+0000] {taskinstance.py:2612} INFO - Dependencies all met for dep_context=non-requeueable deps ti=<TaskInstance: etl_pipeline.validate_data scheduled__2025-02-18T00:00:00+00:00 [queued]>
[2025-02-19T09:38:34.435+0000] {taskinstance.py:2612} INFO - Dependencies all met for dep_context=requeueable deps ti=<TaskInstance: etl_pipeline.validate_data scheduled__2025-02-18T00:00:00+00:00 [queued]>
[2025-02-19T09:38:34.435+0000] {taskinstance.py:2865} INFO - Starting attempt 1 of 3
[2025-02-19T09:38:34.448+0000] {taskinstance.py:2888} INFO - Executing <Task(BranchPythonOperator): validate_data> on 2025-02-18 00:00:00+00:00
[2025-02-19T09:38:34.455+0000] {warnings.py:112} WARNING - /home/***/.local/lib/python3.12/site-packages/***/task/task_runner/standard_task_runner.py:70: DeprecationWarning: This process (pid=91) is multi-threaded, use of fork() may lead to deadlocks in the child.
  pid = os.fork()

[2025-02-19T09:38:34.456+0000] {standard_task_runner.py:72} INFO - Started process 101 to run task
[2025-02-19T09:38:34.457+0000] {standard_task_runner.py:104} INFO - Running: ['***', 'tasks', 'run', 'etl_pipeline', 'validate_data', 'scheduled__2025-02-18T00:00:00+00:00', '--job-id', '66', '--raw', '--subdir', 'DAGS_FOLDER/etl_pipeline.py', '--cfg-path', '/tmp/tmpk24euot7']
[2025-02-19T09:38:34.457+0000] {standard_task_runner.py:105} INFO - Job 66: Subtask validate_data
[2025-02-19T09:38:34.497+0000] {task_command.py:467} INFO - Running <TaskInstance: etl_pipeline.validate_data scheduled__2025-02-18T00:00:00+00:00 [running]> on host fb45ff33766c
[2025-02-19T09:38:34.570+0000] {taskinstance.py:3131} INFO - Exporting env vars: AIRFLOW_CTX_DAG_EMAIL='your_email@example.com' AIRFLOW_CTX_DAG_OWNER='***' AIRFLOW_CTX_DAG_ID='etl_pipeline' AIRFLOW_CTX_TASK_ID='validate_data' AIRFLOW_CTX_EXECUTION_DATE='2025-02-18T00:00:00+00:00' AIRFLOW_CTX_TRY_NUMBER='1' AIRFLOW_CTX_DAG_RUN_ID='scheduled__2025-02-18T00:00:00+00:00'
[2025-02-19T09:38:34.571+0000] {logging_mixin.py:190} INFO - Task instance is in running state
[2025-02-19T09:38:34.572+0000] {logging_mixin.py:190} INFO -  Previous state of the Task instance: queued
[2025-02-19T09:38:34.572+0000] {logging_mixin.py:190} INFO - Current task name:validate_data state:running start_date:2025-02-19 09:38:34.429719+00:00
[2025-02-19T09:38:34.572+0000] {logging_mixin.py:190} INFO - Dag name:etl_pipeline and current dag run status:running
[2025-02-19T09:38:34.572+0000] {taskinstance.py:731} INFO - ::endgroup::
[2025-02-19T09:38:34.573+0000] {baseoperator.py:405} WARNING - BranchPythonOperator.execute cannot be called outside TaskInstance!
[2025-02-19T09:38:34.573+0000] {etl_pipeline.py:83} INFO - Validation passed.
[2025-02-19T09:38:34.573+0000] {python.py:240} INFO - Done. Returned value was: load_data
[2025-02-19T09:38:34.574+0000] {branch.py:38} INFO - Branch into load_data
[2025-02-19T09:38:34.574+0000] {skipmixin.py:230} INFO - Following branch ('load_data',)
[2025-02-19T09:38:34.588+0000] {skipmixin.py:278} INFO - Skipping tasks [('send_alert', -1)]
[2025-02-19T09:38:34.625+0000] {taskinstance.py:340} INFO - ::group::Post task execution logs
[2025-02-19T09:38:34.626+0000] {taskinstance.py:352} INFO - Marking task as SUCCESS. dag_id=etl_pipeline, task_id=validate_data, run_id=scheduled__2025-02-18T00:00:00+00:00, execution_date=20250218T000000, start_date=20250219T093834, end_date=20250219T093834
[2025-02-19T09:38:34.636+0000] {logging_mixin.py:190} INFO - Task instance in success state
[2025-02-19T09:38:34.636+0000] {logging_mixin.py:190} INFO -  Previous state of the Task instance: running
[2025-02-19T09:38:34.637+0000] {logging_mixin.py:190} INFO - Dag name:etl_pipeline queued_at:2025-02-19 09:38:17.399182+00:00
[2025-02-19T09:38:34.637+0000] {logging_mixin.py:190} INFO - Task hostname:fb45ff33766c operator:BranchPythonOperator
[2025-02-19T09:38:34.670+0000] {local_task_job_runner.py:266} INFO - Task exited with return code 0
[2025-02-19T09:38:34.699+0000] {taskinstance.py:3900} INFO - 1 downstream tasks scheduled from follow-on schedule check
[2025-02-19T09:38:34.708+0000] {local_task_job_runner.py:245} INFO - ::endgroup::
